virtcontainers/qemu_arch_base.go

```diff
@@ -24,6 +24,9 @@ type qemuArch interface {
 	// disableNestingChecks nesting checks will be ignored
 	disableNestingChecks()
 
+	// nestingChecks return true if nesting checks are enabled, otherwise false
+	nestingChecks() bool
```

Comment:
I would call this function `isNested()` or `isRunningNested()`

Reply:
`nestingChecks` returns true if nesting checks are enabled or not, `isNested()` or `isRunningNested()` sounds more like a fact that we are running in a nested environment or not.

Reply:
Ok I can see that, but this is very confusing that you're using such a function to check determine `DisableModern`. The thing is, we want to disable modern when we run inside a VM, that is when the flag `nestedRun` is true.
Maybe something is wrong with the whole naming of `nestedRun` initially, but I don't expect that we decide `DisableModern` based on the fact that we check or not the nested environment. WDYT ?

Reply:
agree we should rename those functions